src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-cluster.cpp

```diff
@@ -15,37 +15,40 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-#include "webrtc-transport-provider-server.h"
-
-#include <protocols/interaction_model/StatusCode.h>
+#include "webrtc-transport-provider-cluster.h"
 
 #include <app-common/zap-generated/attributes/Accessors.h>
 #include <app-common/zap-generated/cluster-enums.h>
 #include <app-common/zap-generated/cluster-objects.h>
-#include <app/AttributeAccessInterfaceRegistry.h>
-#include <app/CommandHandler.h>
-#include <app/CommandHandlerInterfaceRegistry.h>
-#include <app/EventLogging.h>
-#include <app/reporting/reporting.h>
+#include <app/server-cluster/AttributeListBuilder.h>
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/PlatformManager.h>
+#include <protocols/interaction_model/StatusCode.h>
 
 #include <iterator>
 #include <memory>
 
 using namespace chip;
 using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::WebRTCTransportProvider;
 using chip::Protocols::InteractionModel::Status;
 
-using ICEServerDecodableStruct = chip::app::Clusters::Globals::Structs::ICEServerStruct::DecodableType;
-using WebRTCSessionStruct      = chip::app::Clusters::Globals::Structs::WebRTCSessionStruct::Type;
-using ICECandidateStruct       = chip::app::Clusters::Globals::Structs::ICECandidateStruct::Type;
-using StreamUsageEnum          = chip::app::Clusters::Globals::StreamUsageEnum;
-using WebRTCEndReasonEnum      = chip::app::Clusters::Globals::WebRTCEndReasonEnum;
-
 namespace {
 
-static constexpr uint16_t kMaxSessionId = 65534;
+constexpr uint16_t kMaxSessionId = 65534;
+
+constexpr DataModel::AcceptedCommandEntry kAcceptedCommands[] = {
+    WebRTCTransportProvider::Commands::SolicitOffer::kMetadataEntry,
+    WebRTCTransportProvider::Commands::ProvideOffer::kMetadataEntry,
+    WebRTCTransportProvider::Commands::ProvideAnswer::kMetadataEntry,
+    WebRTCTransportProvider::Commands::ProvideICECandidates::kMetadataEntry,
+    WebRTCTransportProvider::Commands::EndSession::kMetadataEntry,
```

Comment:
```suggestion
    Commands::SolicitOffer::kMetadataEntry,
    Commands::ProvideOffer::kMetadataEntry,
    Commands::ProvideAnswer::kMetadataEntry,
    Commands::ProvideICECandidates::kMetadataEntry,
    Commands::EndSession::kMetadataEntry,
```

I think we have several instances where we can shorten things